app/views/help.scala.html

```diff
@@ -408,7 +409,79 @@ <h3 class="govuk-heading-s">Titles and descriptions that are sensitive to the pu
+            </p>
+            <img src="@routes.Assets.versioned("images/standard-request-review.png")" class="screenshots" alt="Request a metadata review page" />
+            <p class="govuk-body">
+                After you submit your metadata for review, you will see that your metadata review is in progress.
+            </p>
+            <img src="@routes.Assets.versioned("images/standard-review-progress.png")" class="screenshots" alt="Metadata review in progress page" />
+            <p class="govuk-body">
+                You can only continue with the transfer once you receive an email notification with further instructions.  At this stage you will need to log out and wait for the notification (or you may initiate another transfer if you are transferring multiple consignments).
+            </p>
+            <p class="govuk-body">
+                Once your metadata has been reviewed, you will receive a notification depending on one of the following outcomes:
+            </p>
+            <ol class="govuk-body">
+                <li>If issues are found with your metadata:
+                    <p class="govuk-body">
+                        You will receive two emails.
+                    </p>
+                    <p class="govuk-body">
+                        One from a Digital Transfer Advisor with recommendations for making the necessary amendments before submitting for another review.
+                        Another email will be sent with a link to your in progress transfer. This can also be accessed by signing into Transfer Digital Records, selecting the ‘View Transfers’ button and finding the transfer with the corresponding consignment reference. You will see that issues were found with your metadata.
+                    </p>
+                    <img src="@routes.Assets.versioned("images/standard-view-transfers-page.png")" class="screenshots" alt="View transfers page" />
+                    <img src="@routes.Assets.versioned("images/standard-review-issues.png")" class="screenshots" alt="Issues found in metadata page" />
+                    <p class="govuk-body">
+                        Select ‘Continue’ to return to the ‘Record metadata’ page and address the issues with your metadata by selecting the record and updating the form.
+                    </p>
+                    <img src="@routes.Assets.versioned("images/standard-additional-metadata-short.png")" class="screenshots" alt="Additonal metadata page" />
+                    <p class="govuk-body">
+                        Once you have made the necessary changes, you will be prompted to repeat the steps described above to submit your metadata for review by the Digital Archiving team who will check it again and send you an email notification.
+                    </p>
+                </li>
+                <li>If issues are found with your metadata:
```

Comment:
Is this sentence correct? The steps outlined seem to be about if the metadata is OK?